tests/test_mozillaclub.py

```diff
@@ -184,53 +181,44 @@ def test_fetch_empty(self):
         self.assertEqual(len(events), 0)
 
 
-class TestMozillaClubBackendCache(unittest.TestCase):
-    """MozillaClub backend tests using a cache"""
+class TestMozillaClubBackendArchive(TestCaseBackendArchive):
+    """MozillaClub backend tests using an archive"""
 
     def setUp(self):
-        self.tmp_path = tempfile.mkdtemp(prefix='perceval_')
-
-    def tearDown(self):
-        shutil.rmtree(self.tmp_path)
+        super().setUp()
+        self.backend = MozillaClub(MozillaClub_FEED_URL, archive=self.archive)
 
     @httpretty.activate
-    def test_fetch_from_cache(self):
-        """Test whether the cache works"""
-
-        http_requests = configure_http_server()
-
-        # First, we fetch the events from the server, storing them
-        # in a cache
-        cache = Cache(self.tmp_path)
-        mozillaclub = MozillaClub(MozillaClub_FEED_URL, cache=cache)
+    def test_fetch_from_archive(self):
+        """Test whether a list of events is returned from archive"""
 
-        events = [event for event in mozillaclub.fetch()]
-        self.assertEqual(len(http_requests), 1)
+        configure_http_server()
+        self._test_fetch_from_archive()
 
-        # Now, we get the events from the cache.
-        # The contents should be the same and there won't be
-        # any new request to the server
-        cached_events = [event for event in mozillaclub.fetch_from_cache()]
-        self.assertEqual(len(cached_events), len(events))
-        for i in range(0, len(events)):
-            self.assertDictEqual(cached_events[i]['data'], events[i]['data'])
-        self.assertEqual(len(http_requests), 1)  # no more requests done
-
-    def test_fetch_from_empty_cache(self):
-        """Test if there are not any events returned when the cache is empty"""
+    @httpretty.activate
+    def test_empty_cells_from_archive(self):
+        """Test whether empty cells are not include in the json when using the archive"""
 
-        cache = Cache(self.tmp_path)
-        mozillaclub = MozillaClub(MozillaClub_FEED_URL, cache=cache)
-        cached_events = [event for event in mozillaclub.fetch_from_cache()]
-        self.assertEqual(len(cached_events), 0)
+        configure_http_server()
+        self._test_fetch_from_archive()
```

Comment:
I don't see the point of this code. It doesn't test what the tittle of the method implies.